src/store/selectors.js

```diff
@@ -0,0 +1,41 @@
+import { createSelector } from 'reselect'
+
+const selectProducts = state => state.product.selectedProducts
+const selectProductList = state => state.productList.products;
```

Comment:
again this is so confusing naming. so what's the difference between `selectProducts` and `selectProductList`? it just makes it so difficult to read your code